src/application.rs

```diff
@@ -310,7 +412,7 @@ impl Application {
     }
 
     fn create_terminal_task(&mut self) -> (JoinHandle<()>, CancellationToken) {
-        let mut terminal_event_stream = self.ui.terminal.get_stream();
+        let mut terminal_event_stream = TerminalWrapper::get_stream();
```

Comment:
so now stream is singleton?

Reply:
well, it always was de-facto. the ownership of the stream is pretty complex here but the real underlining stream is a singleton in crossterm. I can explain offline if you wish, very long explanation